iceoryx_meta/CMakeLists.txt

```diff
@@ -127,5 +140,6 @@ message("       Build Properties")
 message("          project name..............: " ${CMAKE_PROJECT_NAME})
 message("          c compiler................: " ${CMAKE_C_COMPILER})
 message("          c++ compiler..............: " ${CMAKE_CXX_COMPILER})
+message("          c++ flags.................: " ${CMAKE_CXX_FLAGS})
```

Comment:
this might mislead since we don't set the flags globally anymore. We set it on the target, e.g. with `${ICEORYX_CXX_STANDARD}` or `${ICEORYX_WARNINGS}`